### PR TITLE
German translation - Quick fix

### PR DESCRIPTION
**Pull request by Swardu**
Currently it said that TurtleCoin is funny instead of fun. Germans don't say that something is "fun", replaced it with secure instead and added missing words. I'm not the best German writer, this translation should be fixed because it's Google-translated.

### CODE REVIEW

**Comment by ar-x** [comment]
@encryptedunicorn, @hurrdurrnein, @armitage0 - can one of you review this change. Thanks.

File: _i18n/de.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ index_page:
   banner_is_safe: TurtleCoin ist Sicher
   banner_one_liner: TurtleCoin ist eine sichere, schnelle und einfache Möglichkeit, privat oder geschäftlich Geld zu versenden.
   banner_download: Download
-  banner_builds: Wir haben Anwendungen für Windows, MacOS und Ubuntu Linux. Falls Du den <strong>Quellcode</strong> bevorzugst, kannst du diesen <a href="https://github.com/turtlecoin/turtlecoin/">hier</a> finden. Und wenn Du schon dabei bist, solltest du uns <a href="http://chat.turtlecoin.lol">hier</a> kontaktieren. Denn nach Personen wie Dir suchen wir!
+  banner_builds: Wir haben Anwendungen für Windows, MacOS und Ubuntu Linux. Falls du den <strong>Quellcode</strong> brauchst, kannst du diesen <a href="https://github.com/turtlecoin/turtlecoin/">hier</a> finden. Und wenn du schon dabei bist, solltest du uns <a href="http://chat.turtlecoin.lol">hier</a> kontaktieren. Denn nach Personen wie Dir suchen wir!
```
Comment (hurrdurrnein) [inline _i18n/de.yml]: Changing "Du" to "du" is ok.
But I'd prefer to stay with "bevorzugst".
@armitage0 , @encryptedunicorn , what do you think?

File: _i18n/de.yml [hunk 2]
```diff
@@ -54,7 +54,7 @@ index_page:
   # why section
   why_title: Warum TurtleCoin?
   why_1_title: Schnelle Transaktionen
-  why_1: TurtleCoin erstellt alle 30 Sekunden einen Block - im Gegensatz zu beispielsweise 10 Minuten, wie bei Bitcoin oder BitcoinCash. Dein Geld ist bei TurtleCoin also 20-mal schneller unterwegs.
+  why_1: TurtleCoin erstellt alle 30 Sekunden einen Block - im Gegensatz zu beispielsweise 10 Minuten, wie bei Bitcoin oder BitcoinCash, deswegen ist dein Geld bei TurtleCoin  20-mal schneller unterwegs.
```
Comment (hurrdurrnein) [inline _i18n/de.yml]: I'd rather stay with the original version. Two sentences are generally more understandable than one long sentence.
@armitage0 , @encryptedunicorn , what do you think?

File: _i18n/de.yml [hunk 3]
```diff
@@ -28,14 +28,14 @@ navbar:
 # https://turtlecoin.lol/
 index_page:
   meta:
-    description: TurtleCoin ist eine spaßige Möglichkeit, privat oder geschäftlich Geld zu versenden. TurtleCoin ist eine von der Community entwickelte, quelloffene Kryptowährung, welche unkompliziert zu minen ist und mit verschiedenen Wallets verwendet werden kann.
-    title: TurtleCoin.lol - TurtleCoin Spaßige, Schnelle, und Einfache Kryptowährung.
+    description: TurtleCoin ist eine sichere, schnelle und einfache Möglichkeit, privat oder geschäftlich Geld zu versenden. TurtleCoin ist eine von der Community entwickelte, quelloffene Kryptowährung, welche unkompliziert zu minen ist und mit verschiedenen Wallets verwendet werden kann.
```
Comment (hurrdurrnein) [inline _i18n/de.yml]: I agree to change the "spaßig" and "witzig" to something else. I wanted to do that myself for some time, but never had the drive to do it.
"Sicher" is fine with me.
@armitage0 , @encryptedunicorn , thoughts?

File: _i18n/de.yml [hunk 4]
```diff
@@ -28,14 +28,14 @@ navbar:
...
+    title: TurtleCoin.lol - TurtleCoin sichere, schnelle und einfache Kryptowährung.
```
Comment (hurrdurrnein) [inline _i18n/de.yml]: Agreed to replace "spaßige" with "sichere".
But still write it with uppercase letters, due to the phrase being a title:
"TurtleCoin Sichere, Schnelle und Einfache Kryptowährung."

File: _i18n/de.yml [hunk 5]
```diff
@@ -28,14 +28,14 @@ navbar:
...
 
   # banner section
   banner_is_fast: TurtleCoin ist Schnell
   banner_is_easy: TurtleCoin ist Einfach
   banner_is_safe: TurtleCoin ist Sicher
-  banner_one_liner: TurtleCoin ist eine witzige Möglichkeit, privat oder geschäftlich Geld zu versenden.
+  banner_one_liner: TurtleCoin ist eine sichere, schnelle und einfache Möglichkeit, privat oder geschäftlich Geld zu versenden.
```
Comment (hurrdurrnein) [inline _i18n/de.yml]: ok

File: _i18n/de.yml [hunk 6]
```diff
@@ -56,7 +56,7 @@ index_page:
   why_1_title: Schnelle Transaktionen
   why_1: TurtleCoin erstellt alle 30 Sekunden einen Block - im Gegensatz zu beispielsweise 10 Minuten, wie bei Bitcoin oder BitcoinCash. Dein Geld ist bei TurtleCoin also 20-mal schneller unterwegs.
   why_2_title: Privatsphäre
-  why_2: TurtleCoin verwendet die gleichen Privätsphärefunktionen wie Monero und Aeon. Jede Transaktion ist vertraulich, außer Du selbst entscheidest Dich dazu, sie zu veröffentlichen.
+  why_2: TurtleCoin verwendet die gleichen Privätsphärefunktionen wie Monero und Aeon. Jede Transaktion ist vertraulich, außer du selbst entscheidest dich dazu, sie zu veröffentlichen.
```
Comment (hurrdurrnein) [inline _i18n/de.yml]: ok

File: _i18n/de.yml [hunk 7]
```diff
@@ -54,9 +54,9 @@ index_page:
   # why section
   why_title: Warum TurtleCoin?
   why_1_title: Schnelle Transaktionen
-  why_1: TurtleCoin erstellt alle 30 Sekunden einen Block - im Gegensatz zu beispielsweise 10 Minuten, wie bei Bitcoin oder BitcoinCash. Dein Geld ist bei TurtleCoin also 20-mal schneller unterwegs.
+  why_1: TurtleCoin erstellt alle 30 Sekunden einen Block - im Gegensatz zu beispielsweise 10 Minuten, wie bei Bitcoin oder BitcoinCash, deswegen ist dein Geld bei TurtleCoin  20-mal schneller unterwegs.
   why_2_title: Privatsphäre
-  why_2: TurtleCoin verwendet die gleichen Privätsphärefunktionen wie Monero und Aeon. Jede Transaktion ist vertraulich, außer Du selbst entscheidest Dich dazu, sie zu veröffentlichen.
+  why_2: TurtleCoin verwendet die gleichen Privätsphärefunktionen wie Monero und Aeon. Jede Transaktion ist vertraulich, außer du selbst entscheidest dich dazu, sie zu veröffentlichen.
```
Comment (hurrdurrnein) [inline _i18n/de.yml]: ok

**Pull request merged**